s29986-PrzemyslawKosiedowski/zajecia2/zadanie_domowe.txt

```diff
@@ -0,0 +1,36 @@
+Zadanie domowe
+Przygotuj skrypt, który odczyta plik (jego nazwa ma być argumentem skryptu) i
+wypisze go na terminalu tak, aby wszystkie wystąpienia Twojego imienia były
+podświetlone na czerwono.
+===============================================================================
+
+Skrypt:
+#!/bin/bash
+plik="$1"
+sed -E "s/Przemysław/\x1B[31m&\x1B[0m/g" "$plik"
+
+Wynik:
+Przemysław ... mango, banan, słoma, Przemysław ... nie potrafię wymyślać
```

Comment:
W Przemyślu, Przemysł awiacyjny nie jest specjalnie rozwinięty. Dlatego Przemysław postanowił, że w Przemyślu po przemyśleniu sprawy uruchomi punkt Przemysław Air, którego Przemysław będzie założycielem, dyrektorem i pracownikiem. Plan wypalił. Klientów było dużo. Dużo do tego stopnia, że Przemysław potrzebował jednak ław dla swoich klientów w poczekalni.
Po pracy Przemysław ławiał ryby w pobliskim jeziorze. Pomoże to to nie jest ale ryby są. Czasami nawet zostawiał niektóre złowione ryby. A gdy wracał to były śniardwe.
Przemysław przemyślał swoje postępowanie i stwierdził, że bedzie lepiej dobierał wolumen połowu do swoich potrzeb by nie marnować zasobów. Gdy doszedł do tego wniosku, tak go on zaskoczył, że wpadł do wody. I do brzegu musiał Przmysław w pław się udać.